### ISSUE TITLE
Ignore shot when name is incorrect in straw.json

### ISSUE THREAD

**Issue by sergicontre** [cellsjs/piscosour#52]
When the name of shot in straw.json is incorrect, the stout no print anything.

**Comment by sbonacho** [comment]
Now i'm getting this error when this happend

```
[16:38:55] Your repository type is: [ recipe ]
[16:38:55] Sipping straw [ test ] shots: [ 'tests2', 'test:1', 'test:2', 'testint' ]
TypeError: Cannot read property 'default' of undefined
    at getShot (/Users/sbonacho/WebstormProjects/piscosour/lib/config.js:138:30)
    at Object.getStrawParams (/Users/sbonacho/WebstormProjects/piscosour/lib/config.js:176:22)
    at synchro (/Users/sbonacho/WebstormProjects/piscosour/lib/sipper.js:58:44)
    at Object.execute (/Users/sbonacho/WebstormProjects/piscosour/lib/sipper.js:26:9)
    at /Users/sbonacho/WebstormProjects/piscosour/lib/sour.js:68:36
    at Object.gush (/Users/sbonacho/WebstormProjects/piscosour/lib/sour.js:36:23)
    at Object.Piscosour.gush (/Users/sbonacho/WebstormProjects/piscosour/index.js:39:21)
    at Object.<anonymous> (/Users/sbonacho/WebstormProjects/piscosour/bin/pisco.js:2:15)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)

Uncatched error:

 TypeError: Cannot read property 'default' of undefined
    at getShot (/Users/sbonacho/WebstormProjects/piscosour/lib/config.js:138:30)
    at Object.getStrawParams (/Users/sbonacho/WebstormProjects/piscosour/lib/config.js:176:22)
    at synchro (/Users/sbonacho/WebstormProjects/piscosour/lib/sipper.js:58:44)
    at Object.execute (/Users/sbonacho/WebstormProjects/piscosour/lib/sipper.js:26:9)
    at /Users/sbonacho/WebstormProjects/piscosour/lib/sour.js:68:36
    at Object.gush (/Users/sbonacho/WebstormProjects/piscosour/lib/sour.js:36:23)
    at Object.Piscosour.gush (/Users/sbonacho/WebstormProjects/piscosour/index.js:39:21)
    at Object.<anonymous> (/Users/sbonacho/WebstormProjects/piscosour/bin/pisco.js:2:15)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
```

i'm going to improve this

**Comment by sbonacho** [comment]
fix with #61
